Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -194,40 +194,46 @@ ErrorOr<void> Layer::try_set_bitmaps(NonnullRefPtr<Gfx::Bitmap> content, RefPtr<
     return {};
 }
 
-void Layer::flip(Gfx::Orientation orientation)
+ErrorOr<void> Layer::flip(Gfx::Orientation orientation)
 {
-    m_content_bitmap = *m_content_bitmap->flipped(orientation).release_value_but_fixme_should_propagate_errors();
+    m_content_bitmap = *TRY(m_content_bitmap->flipped(orientation));
```

Comment:
Possibly a bit of a nitpick but in the case where there is a mask bitmap and the second try operation fails, the layer will be in an inconsistent state. I would suggest something like this:
```
   auto flipped_content_bitmap = TRY(m_content_bitmap->flipped(orientation));
    if (m_mask_bitmap)
        m_mask_bitmap = *TRY(m_mask_bitmap->flipped(orientation));
        
    m_content_bitmap = move(flipped_content_bitmap);
    did_modify_bitmap();
```

Reply:
ty! I have applied your suggestion